code/modules/recycling/conveyor.dm

```diff
@@ -99,26 +127,36 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 		if(SOUTHWEST)
 			forwards = WEST
 			backwards = NORTH
-	if(verted == -1)
+
+	if(inverted)
+		var/temp = forwards
+		forwards = backwards
+		backwards = temp
+	if(flipped) // We need to do this this way to ensure good functionality on corner belts.
```

Comment:
expand your thesis here, why does doing it this way work better for corners?

Reply:
i.e the comment, not reply to my review

Reply:
Right, right.